src/internal_math.rs

```diff
@@ -60,7 +60,7 @@ impl Barrett {
         let x = (((z as u128) * (self.im as u128)) >> 64) as u64;
         let mut v = (z - x * self._m as u64) as u32;
         if self._m <= v {
-            v += self._m;
+            v.wrapping_add(self._m);
```

Comment:
`wrapping_add` doesn't mutate `v` itself; you might want to write:
```suggestion
            v = v.wrapping_add(self._m);
```

Reply:
I was fool, fixed.